seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/config/SourceConfig.java

```diff
@@ -91,9 +93,9 @@ public SourceConfig(
                     .defaultValue(DEFAULT_SCAN_MEM_LIMIT)
                     .withDescription("Memory byte limit for a single query");
 
-    public static final Option<String> STARROCKS_SCAN_CONFIG_PREFIX =
+    public static final Option<Map<String, String>> STARROCKS_SCAN_CONFIG_PREFIX =
             Options.key("scan.params.")
-                    .stringType()
+                    .type(new TypeReference<Map<String, String>>() {})
                     .noDefaultValue()
                     .withDescription("The parameter of the scan data from be");
```

Comment:
why change this? I think we should add test case for this type config.

Reply:
I think we can directly set STARROCKS_SCAN_CONFIG_PREFIX to map
type, instead of setting it to string type, and then parsing it through TypesafeConfigUtils#extractSubConfig() to get the map type, and the TypesafeConfigUtils#extractSubConfig() method has been marked as Deprecated

Reply:
> why change this? I think we should add test case for this type config.

I think we can directly set STARROCKS_SCAN_CONFIG_PREFIX to map
type, instead of setting it to string type, and then parsing it through TypesafeConfigUtils#extractSubConfig() to get the map type, and the TypesafeConfigUtils#extractSubConfig() method has been marked as Deprecated